app/controllers/site.rb

```diff
@@ -21,17 +21,26 @@ class Site < Base
 						end
 					end.compact
 
-					slim :index, locals: { sites: policy_scope(Tint::Site), git_providers: git_providers }
+					sites = policy_scope(Tint::Site)
+					repos = sites.map { |site| site.remote }
+
+					slim :index, locals: { sites: sites, repos: repos, git_providers: git_providers }
 				end
 			end
 
 			post "/" do
 				authorize Tint::Site, :create?
 
+				if params[:provider] == "github"
+					identity = DB[:identities][user_id: pundit_user[:user_id], provider: "github"]
```

Comment:
could also make the `Github` class take in a whole identity record instead of just a token, and do the extraction there.  It *is* github-specific logic